catalyst-gateway/event-db/Earthfile

```diff
@@ -4,6 +4,26 @@
 
 VERSION 0.7
 
+# cspell: words pgdg eventdb dbname 
+
+ADD_PSQL:
```

Comment:
So you won't need this, you will just `FROM +postgress-base` when you need to use that container to run stuff inside it,  or you can run the container to give you a postgres server for testing purposes.